compiler/src/dmd/expressionsem.d

```diff
@@ -15076,21 +15189,30 @@ Expression resolveLoc(Expression exp, const ref Loc loc, Scope* sc)
         return e.expressionSemantic(sc);
     }
 
-    Expression visitLineInit(LineInitExp _)
+    Expression visitLineInit(LineInitExp exp)
     {
+        if (sc.inDefaultArg)
+            return exp;
```

Comment:
This patterns is repeated quite a lot, can that be avoided?

Reply:
Good idea. This can be done once at the start of resolveLoc.